lib/rules/default-props-match-prop-types.js

```diff
@@ -0,0 +1,627 @@
+/**
+ * @fileOverview Enforce all defaultProps are defined in propTypes
+ * @author Vitor Balocco
+ * @author Roy Sutton
+ */
+'use strict';
+
+var has = require('has');
+var find = require('array.prototype.find');
```

Comment:
What's sad is I originally implemented with `Array.find()` but realized this didn't match with the current style.